docs/reference/expectations/result_format.md

```diff
@@ -4,66 +4,62 @@ title: Result format
 
 
 The `result_format` parameter may be either a string or a dictionary which specifies the fields to return in `result`.
-  * For string usage, see `result_format` values.
+  * For string usage, see `result_format` [values and fields](#result_format-values-and-fields).
   * For dictionary usage, `result_format` which may include the following keys:
```

Comment:
This item doesn't read well -- the word "which" makes `result_format` seem insufficient of a subject.

I would change it to:
```
  * For dictionary usage, `result_format` may include the following keys:
```
or:
```
  * For dictionary usage, provide `result_format` as a dictionary, which may include the following keys:
```

(I think I prefer the first of the two.)

Reply:
Done. Thanks for the catch :)